### DISCUSSION TITLE
[Feature Request] Slowmotion / Speedup Video

### DISCUSSION THREAD

**Discussion by JSchimmelpfennig** [mifi/lossless-cut#638] [Ideas]
Hi guys,
first of all - I really love losslesscut!
I would like to see an option to speed up or slow down the video. I have no idea if that's easy to implement, but I would be very happy because that would be great to do a fast timeleapse (32x speed) or just slowmotion (1/4 speed) e.g.

Thank you!

**Comment by Shakil-Shahadat** [comment]
See #586.
